src/utils/calldata/validate.ts

```diff
@@ -33,6 +33,15 @@ import {
   isTypeUint,
 } from './cairo';
 
+/**
```

Comment:
JSDoc not needed since the function is not exported

all of the below actually